packages/scandipwa/src/component/Popup/Popup.component.js

```diff
@@ -152,6 +153,7 @@ export class Popup extends Overlay {
                         { this.renderTitle() }
                         { this.renderCloseButton() }
                     </header>
+                    <NotificationList />
```

Comment:
Since the notification should be rendered only on mobile devices, please check the `isMobile` prop and render it only if it's true.